### PR TITLE
Lberg/tests fixtures

### PR DESCRIPTION
**Pull request by lucabergamini**
Add `conftest.py` for fixtures. It avoids writing the same fixtures all over the code.

### CODE REVIEW

File: l5kit/l5kit/tests/evaluation/compute_mse_loss_test.py [hunk 1]
```diff
@@ -7,21 +7,19 @@
...
     data_fake = ChunkedDataset(str(tmp_path))
-    data_fake.scenes = np.asarray(data.scenes).copy()
-    data_fake.frames = np.asarray(data.frames).copy()
-    data_fake.agents = np.asarray(data.agents).copy()
+    data_fake.scenes = np.asarray(zarr_dataset.scenes).copy()
+    data_fake.frames = np.asarray(zarr_dataset.frames).copy()
+    data_fake.agents = np.asarray(zarr_dataset.agents).copy()
     data_fake.agents["centroid"] += np.random.rand(*data_fake.agents["centroid"].shape) * 1e-2
 
-    export_zarr_to_ground_truth_csv(data_fake, str(tmp_path / "gt3.csv"), 0, 50, 0.5)
```
Comment (oliver-scheel) [inline l5kit/l5kit/tests/evaluation/compute_mse_loss_test.py:24]: Switch from 0 to 10 on purpose?

File: l5kit/l5kit/tests/sampling/deep_prediction_test.py [hunk 2]
```diff
@@ -4,25 +4,11 @@
 import numpy as np
 import pytest
 
-from l5kit.configs import load_config_data
 from l5kit.data import AGENT_DTYPE, FRAME_DTYPE, ChunkedDataset
 from l5kit.rasterization import StubRasterizer
 from l5kit.sampling import generate_agent_sample
```
Comment (oliver-scheel) [inline l5kit/l5kit/tests/sampling/deep_prediction_test.py:11]: These were never used in the previous version?

File: l5kit/l5kit/tests/evaluation/compute_mse_loss_test.py (hunk 1 above)
Comment (lucabergamini) [reply l5kit/l5kit/tests/evaluation/compute_mse_loss_test.py:24]: yeah, it speeds tests up :)

File: l5kit/l5kit/tests/sampling/deep_prediction_test.py (hunk 2 above)
Comment (lucabergamini) [reply l5kit/l5kit/tests/sampling/deep_prediction_test.py:11]: yeah it was used to get a fixture here on line 20

**Pull request merged**